std/random.d

```diff
@@ -1281,17 +1294,79 @@ A single unsigned integer seed value, different on each successive call
 */
 @property uint unpredictableSeed() @trusted
```

Comment:
this should be a template with trusted escapes so the attributes can be inferred